src/components/PoolOverview.tsx

```diff
@@ -31,6 +32,17 @@ interface Props {
   onClickMigrate?: (e: React.MouseEvent<HTMLButtonElement>) => void
 }
 
+const LinkStyle = styled(Link)(({ theme }) => ({
```

Comment:
LinkStyled or StyledLink or find a better naming convention for styled components.